src/vs/editor/contrib/codeAction/browser/codeActionContributions.ts

```diff
@@ -33,3 +33,15 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		},
 	}
 });
+
+Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
+	...editorConfigurationBaseNode,
+	properties: {
+		'editor.codeActionWidget.showInlineQuickfixes': {
```

Comment:
`inline` in the name confused me. Maybe something like `includeNearbyQuickFixes`